src/components/Skeleton/Skeleton.scss

```diff
@@ -46,6 +46,12 @@
 }
 
 .monday-style-skeleton {
+  display: flex;
+
+  .full-width {
```

Comment:
"full-width" is very generic class name, the danger of using a very generic class name is that we can receive by mistake styles which defined by other developers in the monolith which will affect our skeleton.
@orrgottlieb suggest that we will create a new scss modules file for the skeleton and add only this class to there for avoiding conflicits. Wdty?

Reply:
Moved it to css modules 👍